components/monitoring/cpu/cpu.html

```diff
@@ -37,16 +37,15 @@
 
     <div class="col l6 m12">
         <div class="card">
-            <div class="card-content">
+            <div class="card-content" id="cpuTemperature">
                 <span class="card-title">CPU Temperature</span>
                 <canvas id="canvas-cpu-temperature"></canvas>
             </div>
         </div>
-    </div>
-
+</div>
```

Comment:
Could you change this line back to how it was before? I don't think this kind of indentation is correct.

Reply:
sure! it's done